### DISCUSSION TITLE
Status of project

### DISCUSSION THREAD

**Discussion by IllustratedMan-code** [guile-rs/guile-rs#11] [Ideas]
I think the idea behind this project has a lot of potential! 

Is it usable currently? What needs to be done? Is there a planned trajectory of the project?

I would love to help out, specifically in terms of examples and documentation.

**Comment by IllustratedMan-code** [comment]
Some sort of feature list and example in the readme would be a good start I think.

**Comment by shymega** [comment]
Just moving this to the discussions section, then we can talk in more detail.

**Comment by shymega** [comment]
Right, had some time to reply.

Honestly, it's not progressing much. I need to work with other maintainers to make a unified approach. FFI on these bindings are a bit tricky. They are hacky, and Guile is hacky as well.

There's a more polished version on GitLab, but I do not use GitLab, and that could be an issue...

**Reply by aryalaadi** [reply]
link to the more polished version on gitlab?
